templates/article/article.js

```diff
@@ -136,3 +141,15 @@ function createSocialMediaButtons() {
   decorateIcons(socialMediaButtons);
   return socialMediaButtons;
 }
+
+async function createMagazineFooter() {
+  const issue = getMetadata('issue').toLowerCase();
+  const summary = await fetch(`/navigation/magazine-summary/${issue}.plain.html`);
```

Comment:
maybe apply toClassName to the issue, I am not sure users will always have the case and spacing right.